gammapy/estimators/map/tests/test_core.py

```diff
@@ -490,3 +490,6 @@ def test_slice_by_coord():
 
     sliced_map2 = ref_map.slice_by_coord({"energy": slice(0.5 * u.TeV, 5.0 * u.TeV)})
     assert sliced_map2.geom.axes["energy"].nbin == 1
+
+    sliced_map3 = ref_map.slice_by_coord({"energy": slice(3.0 * u.TeV, 11.0 * u.TeV)})
```

Comment:
```suggestion
    sliced_map3 = ref_map.slice_by_coord({"energy": slice(1.0 * u.TeV, 10.0 * u.TeV)})
```

Reply:
testing on exact edges might be useful since `sliced_map2` already tests the other case

Reply:
why don't you add a test for the slicing on time in an analogous fashion to the energy?